File: Sem5Task36/Program.py
import random
list = []  #Создаем пустой список
count = 0
num = int(input())   #вводим размер массива
for i in range(num):
    rnd_list = random.randrange(100, 999)   #создаем числа 
    list.append(rnd_list)  #вставляем их в список 
for i in range(1,len(list),2):  #проходимся по каждому четному индексу
    count+=list[i]    #складываем
print(list,count)




#36*
#Возможно я не так понял задачу, тут я нахожу колличество одинаковых пар
num= input().split()  #вводим список
counter=0
for i in range(len(num)):  #первый цикл на длину списка
    for j in range(i+1, len(num)):  #второй цикл на длину списка '''прибавил i+1, чтобы не сравнивать с самим собой'''
        if num[i]==num[j]:  #сравниваем 
            counter+=1  #добавлем в счетсчик
print(counter)   #выводим

File: Sem5Task36/Program.cs
int ReadData(string msg)
{
    Console.WriteLine(msg);
    return int.Parse(Console.ReadLine() ?? "0");
}

void PrintData(string res)
{
    Console.WriteLine(res);
}

int[] Gen1DArr(int num, int begin, int end)
{
    Random rnd = new Random();
    int[] arr = new int[num];
    for (int i = 0; i < arr.Length; i++)
    {
        arr[i] = rnd.Next(begin, end);
    }
    return arr;
}

void Print1DArr(int[] arr)
{
    Console.Write("[");
    for (int i = 0; i < arr.Length - 1; i++)
    {
        Console.Write(arr[i] + ", ");
    }
    Console.WriteLine(arr[arr.Length - 1] + "]");
}

int CountElem(int[] arr)
{
    int res = 0;
    for (int i = 1; i < arr.Length; i+=2)
    {
        res+=arr[i];
    }
    return res;
}


int arrLen = ReadData("Введите длину массива.");
int begin = ReadData("Введите первую границу диапозона чисел: ");
int end = ReadData("Введите вторую границу диапозона чисел: ");
int[] testArr = Gen1DArr(arrLen,begin,end);
Print1DArr(testArr);
int count = CountElem(testArr);
PrintData("Сумма числе в четных позициях массива :"+count);